specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,687 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview data policies REST API Document",
+    "version": "2023-08-01-preview"
+  },
+  "paths": {
+    "/data-policies": {
+      "get": {
+        "tags": [
+          "DataPolicy_List"
+        ],
+        "summary": "API to list the purview data policies. Returns policies, both in draft and published states",
+        "description": "List all data policies.",
+        "operationId": "DataPolicy_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy service",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        },
+        "x-ms-examples": {
+          "DataPolicy_List": {
+            "$ref": "./examples/DataPolicy_List.json"
+          }
+        }
+      }
+    },
+    "/data-policies/{policyName}": {
+      "put": {
+        "tags": [
+          "DataPolicy_CreateOrReplace"
+        ],
+        "description": "Create or replace a data policy in draft state. The policy will go into draft state and will not be active until it is published.",
+        "summary": "API to create or replace a purview data policy in draft state.",
+        "operationId": "DataPolicy_CreateOrReplace",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/policyName"
+          },
+          {
+            "$ref": "#/parameters/if-match"
+          },
```

Comment:
Why it has only `if-match`?

E.g. If I want to create a resource but don't want it overwrite an existing one (in possible concurrency situation), I would like to use `if-none-match: *` here.

Reply:
In purview policy we only support create and full replace of a policy. We do not support `if-none-match: *` in this version.